fdb-relational-api/src/main/java/com/apple/foundationdb/relational/api/Options.java

```diff
@@ -38,7 +40,8 @@
 import java.util.Map;
 import java.util.Objects;
 
-public final class Options {
+@API(API.Status.EXPERIMENTAL)
+public class Options {
```

Comment:
I'm not necessarily opposed to removing the `final` here, but it does seem like a bit of a separate change. I'm not sure the `final`ity of our classes was particularly thoughtfully considered, but final-by-default is probably closer to what we want than the other way around